guides/common/modules/proc_enabling-the-templatesync-plugin.adoc

```diff
@@ -9,3 +9,5 @@
 # {foreman-installer} --enable-foreman-plugin-templates
 ----
 . To verify that the plugin is installed correctly, ensure *Administer* > *Settings* includes the *TemplateSync* menu.
```

Comment:
```suggestion
. To verify that the plugin is installed correctly, ensure *Administer* > *Settings* includes the *Template Sync* menu.
```

out of scope, but please:
![image](https://github.com/user-attachments/assets/389bd24a-64d1-4e14-bcf1-814923af1205)

tested on Foreman 3.11

Reply:
You're right but there are more occurrences of TemplateSync in the repo than just these. How about we file an issue to get them all fixed in a separate PR?

Reply:
https://github.com/theforeman/foreman-documentation/issues/3400